### ISSUE TITLE
configurable update-cache path

### ISSUE THREAD

**Issue by SFJohnson24** [cdisc-org/cdisc-rules-engine#1417]
the default behavior is relative to where core.py is executed but there should be an optional argument for a path to update the cache at that directory path that is given by the user

**Comment by tomhub** [comment]
App 
```python:
from platformdirs import user_cache_dir
from pathlib import Path

APP_NAME = "CDISC-RULES-ENGINE"
APP_AUTHOR = "CDISC-ORG"  # Optional, but recommended on Windows/macOS

# Get the cross-platform cache directory path
cache_dir_str = user_cache_dir(
    appname=APP_NAME, 
    appauthor=APP_AUTHOR, 
    # Optionally, set version to create a versioned cache path
    # version="1.0" 
)

# Convert the string path to a modern Path object for easy use
cache_path = Path(cache_dir_str)

# Ensure the directory exists before using it
cache_path.mkdir(parents=True, exist_ok=True)

print(f"Using cache directory: {cache_path}")
```

This should give OS path to app's cache dir in OS expected location. Hopefully this helps.

Although, I think it would be good to pass a config file (yaml/json/toml etc.) to cdisc-core-engine with appropriate user settings (dir locations, api keys, etc.).
